src/Symfony/Component/Security/Http/EventListener/SessionListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Security\Http\Event\LoginSuccessEvent;
+use Symfony\Component\Security\Http\Session\SessionAuthenticationStrategyInterface;
+
+/**
+ * Migrates the session after successful login.
```

Comment:
Maybe:

> Migrates the session after successful login.
>
> Should be registered as a subscriber to any "stateful" firewalls.

Not sure - it's just that you can't see (from inside this class) that this is not being
registered on stateless firewalls.